_global/js/board_front.js

```diff
@@ -138,6 +152,11 @@ function parseGrowthTags(){
                     {
                         newGrowthCellHTML += `${openTag}{reclaim-one}<growth-text>Reclaim One</growth-text></growth-cell>`
                         break;
+                    }
+				case 'reclaim-none':
+                    {
+                        newGrowthCellHTML += `${openTag}{reclaim-none}<growth-text>Reclaim None</growth-text></growth-cell>`
+                        break;
```

Comment:
I shouldn't have committed this change - its a joke icon for my Water Acts Not spirit.
Can you reject specific lines?

Reply:
No. You can either go back in your tree and modify the old commit, then force-push to your own branch. Alternatively (and this is easier) just add another commit on top removing these lines again.